frontend/src/app/offers/offer-detail/offer-detail.component.html

```diff
@@ -65,3 +65,6 @@ <h3 class="card-header">Organizator</h3>
          </div>
     </div>
 </div>
+<ng-template #notFound>
```

Comment:
I might be wrong, but won't this "Not found" display while angular is actually fetching the offer? So there will always be "not found" blinking on the page?

Reply:
I didn't think about that. I wanted to simply redirect to 404, but then thought about user experience - and for good of that we should have the same path user tried get to, without changing to 404. Didn't know how to achieve that with offers and organizations, so I tried with ngIf, but you're right, it would be blinking. I'll have to change this.